<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JuegoTurnos</class>
 <widget class="QMainWindow" name="JuegoTurnos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Juego Multijugador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QGroupBox" name="questionGroup">
      <property name="title">
       <string>Pregunta</string>
      </property>
      <layout class="QVBoxLayout" name="questionLayout">
       <item>
        <widget class="QLabel" name="playerTurnLabel">
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="questionImageLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="questionTextLabel">
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="optionsLayout">
         <item>
          <widget class="QPushButton" name="optionAButton"/>
         </item>
         <item>
          <widget class="QPushButton" name="optionBButton"/>
         </item>
         <item>
          <widget class="QPushButton" name="optionCButton"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="feedbackLabel">
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="finishButton">
      <property name="text">
       <string>Terminar Partida</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="rankingGroup">
      <property name="title">
       <string>Ranking</string>
      </property>
      <layout class="QVBoxLayout" name="rankingLayout">
       <item>
        <widget class="QTableView" name="rankingTableView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
